var/spack/repos/builtin/packages/libev/package.py

```diff
@@ -11,7 +11,7 @@ class Libev(AutotoolsPackage):
     after libevent, but without its limitations and bugs."""
 
     homepage = "http://software.schmorp.de/pkg/libev.html"
-    url      = "http://dist.schmorp.de/libev/libev-4.24.tar.gz"
+    url      = "http://dist.schmorp.de/libev/Attic/libev-4.24.tar.gz"
```

Comment:
This is automatically handled by `list_url`:
```console
$ spack fetch libev
==> Fetching http://dist.schmorp.de/libev/libev-4.24.tar.gz
#=#=- #      #                                                                                                                                                           
curl: (22) The requested URL returned error: 404 Not Found
==> Fetching from http://dist.schmorp.de/libev/libev-4.24.tar.gz failed.
==> Fetching http://dist.schmorp.de/libev/Attic/libev-4.24.tar.gz
################################################################### 100.0%
```

Reply:
But "spack install libev" can not work well..
So, any suggestion for solving this  issue?
https://github.com/spack/spack/issues/13937

Reply:
Maybe it's just a network problem for me.

Reply:
That's strange. I don't know why it works for me but not for you. Since the latest version is also in the `Attic`, I'm fine with moving all downloads to the `Attic`. Can you also delete the `list_url` below? It's no longer needed.